src/main/java/net/arcaniax/buildersutilities/utils/Items.java

```diff
@@ -41,6 +41,16 @@
 
 public class Items {
 
+    public static ItemStack create(String materialName, String name, String lore) {
+        Material mat = Material.getMaterial(materialName);
+
+        if(mat == null) {
+            return null;
+        }
+
+        return Items.create(mat, (short) 0, 1, name, lore);
+    }
+
```

Comment:
Unneeded; outsourcing private classes to public methods serve no purpose in this case.

Reply:
If i understand, i may add this logic into `SecretBlockMenuProvider.java` ?
Because ItemStack constructor did not allow nullable variable for Material ?